Gemfile

```diff
@@ -37,8 +34,7 @@ gem 'kaminari'
 gem 'oj'
 
 # Setup dependencies
-gem 'exercism-config', '>= 0.86.0'
-# gem 'exercism-config', path: '../exercism_config'
+gem 'exercism-config', '>= 0.86.0' # gem 'exercism-config', path: '../exercism_config'
```

Comment:
This is rubocop. I'm happy to revert if you'd like me to

Reply:
```suggestion
gem 'exercism-config', '>= 0.86.0' # path: '../exercism_config'
```